asdf_astropy/converters/slicedwcs/slicedwcs.py

```diff
@@ -0,0 +1,34 @@
+from asdf.extension import Converter
+
+
+class SlicedWCSConverter(Converter):
+    tags = ("tag:astropy.org:astropy/slicedwcs/slicedwcs-*",)
+    types = ("astropy.wcs.wcsapi.wrappers.sliced_wcs.SlicedLowLevelWCS",)
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from astropy.wcs.wcsapi.wrappers.sliced_wcs import SlicedLowLevelWCS
+
+        wcs = node["wcs"]
+        slice_array = []
+        slice_array = [
+            s if isinstance(s, int) else slice(s["start"], s["stop"], s["step"]) for s in node["slices_array"]
+        ]
+        return SlicedLowLevelWCS(wcs, slice_array)
+
+    def to_yaml_tree(self, sl, tag, ctx):
+        node = {}
+        node["wcs"] = sl._wcs
+        node["slices_array"] = []
+
+        for s in sl._slices_array:
+            if isinstance(s, slice):
```

Comment:
When is this not a slice?

Reply:
If we examine the first test case from [test_slicedwcs.py](https://github.com/astropy/asdf-astropy/pull/235/files#:~:text=wcs0%20%3D%20SlicedLowLevelWCS(wcs%2C%201)), the initial element of the `slices_array` will be an integer:
```
(Pdb) wcs0 = SlicedLowLevelWCS(wcs, 1)
(Pdb) wcs0._slices_array
[1, slice(None, None, None), slice(None, None, None), slice(None, None, None)]
(Pdb)
```

Reply:
Is there a public API for accessing the slices?

Reply:
I don't think there is a public API for accessing the slices.